src/Tickets/Commerce/Cart.php

```diff
@@ -679,4 +624,78 @@ public static function get_cart_hash_cookie_name(): string {
 
 		return sanitize_title( $filtered_cookie_name );
 	}
+
+	/**
+	 * Map the ticket data to a more usable format.
+	 *
+	 * @since TBD
+	 *
+	 * @param array $ticket_data  Array of raw ticket data.
+	 * @param array $tickets_meta Array of ticket meta data.
+	 *
+	 * @return array Array of mapped ticket data.
+	 */
+	protected function map_ticket_data( array $ticket_data, array $tickets_meta ): array {
+		/** @var Tickets_Handler $handler */
+		$handler = tribe( 'tickets.handler' );
+
+		return array_map(
+			static function ( $ticket ) use ( $handler, $tickets_meta ) {
+				// Merge the ticket data with the default ticket data.
+				$ticket = array_merge(
+					[
+						'ticket_id' => 0,
+						'quantity'  => 0,
+						'optout'    => false,
+						'iac'       => 'none',
+						'extra'     => [],
+					],
+					$ticket
+				);
+
+				// Normalize and validate the quantity.
+				$ticket['quantity'] = (int) $ticket['quantity'];
+				if ( $ticket['quantity'] < 1 ) {
+					return false;
+				}
+
+				// Normalize and validate the ticket ID.
+				$ticket['ticket_id'] = (int) $ticket['ticket_id'];
+				if ( true !== $handler->is_ticket_readable( $ticket['ticket_id'] ) ) {
+					return false;
+				}
+
+				// Add attendee data.
+				if ( ! empty( $tickets_meta[ $ticket['ticket_id'] ]['attendees'] ) ) {
+					$ticket['extra']['attendees'] = $tickets_meta[ $ticket['ticket_id'] ]['attendees'];
+				}
+
+				// Add the optout and IAC data.
+				$ticket['extra']['optout'] = tribe_is_truthy( $ticket['optout'] );
+				$ticket['extra']['iac']    = sanitize_text_field( $ticket['iac'] );
+				unset( $ticket['optout'], $ticket['iac'] );
+
+				// Add the ticket object.
+				$ticket['obj'] = Tickets::load_ticket_object( $ticket['ticket_id'] );
+
+				return $ticket;
+			},
+			$ticket_data
+		);
+	}
+
+	/**
+	 * Returns the name of the transient used by the cart.
+	 *
+	 * @since 5.1.9
+	 * @depecated Use `get_transient_key()` via the Cart trait instead.
+	 *
+	 * @param string $id The cart id.
+	 *
+	 * @return string The transient name.
+	 */
+	public static function get_transient_name( $id ) {
+		_deprecated_function( __METHOD__, 'TBD', 'TEC\Tickets\Commerce\Traits\Cart::get_transient_key' );
+		return Commerce::ABBR . '-cart-' . md5( $id ?? '' );
```

Comment:
Should you call the trait here as a passthrough until we remove this method? That way the data stays in sync.

Reply:
@redscar It's just a string concatenation, and the trait does the same thing. I just thought it made more sense for using this where it is needed to put it in a trait rather than relying on static methods.